infrastructure/lamba-mns-notification.tf

```diff
@@ -0,0 +1,97 @@
+module "mns-notification-lambda" {
+  source  = "./modules/lambda"
+  name    = "MNSNotificationLambda"
+  handler = "handlers.mns_notification_handler.lambda_handler"
+  iam_role_policies = [
```

Comment:
This block will need refactoring when [PRMP-1221](https://nhse-dsic.atlassian.net/browse/PRMP-1221) is merged.